platform/linux_platform.go

```diff
@@ -1069,10 +1069,28 @@ func (p linux) SetupLogDir() error {
 }
 
 func (p linux) SetupOptDir() error {
-	optDir := "/var/opt"
+	varOptDir := "/var/opt"
+
+	boshRootVarOptDirPath := path.Join(p.dirProvider.DataDir(), "root_var_opt")
+	err := p.fs.MkdirAll(boshRootVarOptDirPath, userRootOptDirPermissions)
+	if err != nil {
+		return bosherr.WrapError(err, "Creating root var opt dir")
+	}
+
+	_, _, _, err = p.cmdRunner.RunCommand("chown", "root:root", boshRootVarOptDirPath)
+	if err != nil {
+		return bosherr.WrapError(err, "Chowning root var opt dir")
+	}
+
+	err = p.bindMountDir(boshRootVarOptDirPath, varOptDir, true)
```

Comment:
All right for the issue with `noexec` that has been reported in Slack. But we should document here _why_ we do this, so that future maintainers after us will know _how_ they can (ideally) re-test for the workaround still being necessary, and know _when_ they can remove the workaround code.

Reply:
@bgandon What do you think of this comment and the one further down the file?

Reply:
(If you like them, I can squash this into the previous commit and force push.)

Reply:
It's all right yes 👍